builtins-default-costs/src/lib.rs

```diff
@@ -122,6 +123,13 @@ lazy_static! {
             core_bpf_migration_feature: None,
         },
     ),
+    (
+        secp256r1_program::id(),
+        BuiltinCost {
+            native_cost: 0,
+            core_bpf_migration_feature: None,
+        }
+    )
     // DO NOT ADD MORE ENTRIES TO THIS MAP
```

Comment:
Ha apparently this warning wasn't sufficient? We can't add more entries to this map without risking a consensus failure because after https://github.com/anza-xyz/agave/pull/3799, any changes will impact the default compute unit limit we calculate for transactions which don't specify a limit explicitly. The specific issues is that any tx instruction which invokes a "builtin program" (i.e. an entry in this map) will be allocated 3k cu's rather than 200k cu's. If you revert this change the rest of the PR looks ok to me.

Reply:
Yes I did see the warning and asked about it, but I seemed to have misinterpreted the answers that I got 😅. I will revert!

Reply:
Please let's land https://github.com/anza-xyz/agave/pull/3866 and eliminate this footgun.

Reply:
#3948 to document cost model issue with precompile